### PR TITLE
auth fixed, tailwind fixed, routing fixed. svgIcons not getting loaded.....

### PR DESCRIPTION
**Pull request by JAYBORICHA07**

### CODE REVIEW

**Comment by JAYBORICHA07** [comment]
image

**Review by KMJ-007 — APPROVED** [review]
LGTM

File: apps/pwa/vite.config.ts [hunk 1]
```diff
@@ -1,58 +1,33 @@
+import path from "path";
 import react from "@vitejs/plugin-react";
-import { defineConfig, loadEnv } from "vite";
-import { VitePWA } from "vite-plugin-pwa";
```
Comment (krishna-404) [inline apps/pwa/vite.config.ts:3]: why removing `PWA` funcitionality?

File: apps/pwa/tsconfig.json [hunk 2]
```diff
@@ -1,26 +1,44 @@
 {
   "compilerOptions": {
-    "target": "ES2020",
-    "types": ["vite/client"],
+    "target": "ESNext", // Target version of compiled code (ECMAScript).
+    "module": "ESNext", // Type of module system used. ESNext supports more features, such as dynamic imports.
+    "lib": ["ESNext", "DOM", "DOM.Iterable"], // Library files to include. DOM, DOM.Iterable, and ESNext library files are basic dependencies in TypeScript development.
```
Comment (krishna-404) [inline apps/pwa/tsconfig.json:5]: Why changing basic `tsconfig`?

File: apps/pwa/src/layouts/_common/setting-button.tsx [hunk 3]
```diff
@@ -254,50 +254,6 @@ export default function SettingButton() {
...
-              >
-                <div className="flex h-4 w-full items-center gap-1  p-1">
-                  <div
-                    className="h-2 w-2 flex-shrink-0 rounded"
-                    style={{
-                      background: layoutBackground(ThemeLayout.Horizontal),
-                    }}
-                  />
-                  <div
-                    className="h-1 w-4 flex-shrink-0 rounded opacity-50"
```
Comment (krishna-404) [inline apps/pwa/src/layouts/_common/setting-button.tsx:280]: why is this commented code still in the code-base?

File: apps/api/src/auth.router.ts [hunk 4]
```diff
@@ -71,15 +70,14 @@ export const authRouter = (
 
         let user = await getUserByEmail(userInfo.email);
 
-        if (!user) {
+        if (!user[0]?.id) {
```
Comment (krishna-404) [inline apps/api/src/auth.router.ts:73]: why is the user object an array?

**Pull request merged**